sheets/README.md

```diff
@@ -0,0 +1,128 @@
+# xPro Google Sheets
+
+**SECTIONS**
+1. [What Is This?](#what-is-this)
+1. [How It Works](#how-it-works)
+1. [Types of Sheets](#types-of-sheets)
+1. [Sheet Basics](#sheet-basics)
+1. [Handling Request Sheet Errors](#handling-request-sheet-errors)
+1. [Handling Assignment Sheet Errors](#handling-assignment-sheet-errors)
+1. [Development Setup](#development-setup)
+
+## What Is This?
+
+Rather than support web UIs for certain internal Customer Support and Finance tasks, we chose to support these tasks  
+through UIs that are already very familiar to these teams: Google Sheets and Google Forms. 
+Using various Google Drive and Sheets APIs, the following is now possible: 
+
+1. CS/finance users can make various requests (e.g.: enrollment code creation/assignment, enrollment deferrals) by filling 
+  out a Google Form or entering data into Google Sheets.
+1. Those requests are automatically processed by our app.
+1. Users can monitor the status of their requests directly in the spreadsheets.  
+
+## How It Works
+
+The basic workflow for most flavors of xPro sheets is roughly as follows:
+
+1. A user fills out a Google Form. That form submission is a request for some work to 
+be done, and it is automatically added to a form responses worksheet within a relevant spreadsheet.
+    - That submission also appears in the main worksheet within the spreadsheet automatically via a query.
+1. In response to the change in the spreadsheet, Google sends a file watch request to an endpoint in our app, which 
+  indicates to us that the spreadsheet has been changed, so we may have some new/updated requests to handle.
+1. Our app scans the main worksheet of the spreadsheet for new/updated rows and takes the appropriate action if the
+  data is all valid (e.g.: issuing a refund, creating new enrollment codes, etc.)
+1. If a request was completed for some row, or if an error occurred, our app updates certain columns in the 
+  main worksheet to indicate the status of that request.
+
+## Types of Sheets
+
+Right now there are three sheet types:
+
+1. Enrollment Code Request sheet (a.k.a. Coupon Request sheet)
+1. Enrollment Code Assignment sheets (which are created based on Enrollment Code Request sheet entries)
+1. Change of Enrollment Request sheet. This includes worksheets for the following:
+    1. Refund Requests
+    1. Deferral Requests
+
+New requests for the "request"-type sheets are submitted via Google Form:
+
+1. [Enrollment Code Request form](https://docs.google.com/forms/d/1XfV5ppOYCFHd2Z1B1uL75C-ChgytHfkTD8YOSQM0vFM/edit) 
+1. [Refund Request form](https://docs.google.com/forms/d/1n-TYFd91zKkSkjjudoACMId-mfUeAQ-rGMCzTKqPVFQ/edit)
+1. [Deferral Request form](https://docs.google.com/forms/d/1lEbKGfLxMwh0Drj-CQJo5QXU2evEqxEP5iFGB4qNrw8/edit)
```

Comment:
@briangrossman alerted me that these forms _cannot_ be set to require authentication. So I think we should remove the links from here. 

```suggestion
1. Enrollment Code Request form
1. Refund Request form
1. Deferral Request form
```